test/vast_tracker.js

```diff
@@ -130,6 +130,25 @@ describe('VASTTracker', function () {
         });
       });
 
+      describe('#setProgress trackOnce', () => {
+
+        after(()=> {
+          this.stub?.restore();
+        })
+
+        it('should send firstQuartile event with trackOnce : false', () => {
+          this.stub = sinon.stub(this.Tracker, 'track').callsFake((eventName, params) => {
+            console.log("testprogress : "+eventName);
+            if(eventName === 'progress-23') {
+              params.once.should.equal(false);
+              
+            }
+          });
+
+          this.Tracker.setProgress(23, null, false);
+        });
+      });
+
```

Comment:
Thanks you for adding the test case.

Since we will migrate all mocha tests to Jest in a future realease based on the modularization of the vast-client (https://github.com/dailymotion/vast-client-js/pull/458) you can discard (https://github.com/dailymotion/vast-client-js/pull/461/commits/a3c1690c0ebf58204312574075a990af808debab).

I will add it directly to my branch that will be merged after yours.

Sorry for the inconvenience.